discovery-provider/src/queries/confirm_indexing_transaction_error.py

```diff
@@ -7,7 +7,7 @@
 logger = logging.getLogger(__name__)
 
 # Percent of total discovery nodes needed to skip indexing a transaction
-INDEXING_FAILURE_CONSENSUS_PERCENT = 1
+INDEXING_FAILURE_CONSENSUS_PERCENT = .9
```

Comment:
is this change for performance or some other reason?

Reply:
just since some nodes might be unhealthy or down

Reply:
I think I said 80% on the call, 90% is good too -- we should try to inform our decision-making here with some data. If a node is down, they won't be able to count towards consensus. On a bad day, how many nodes are down that would count against our tally?

Reply:
yeah i kinda wanted to set a high standard at first and loosen if needed but it probably is too high. let's say on a bad day 10% of the nodes are down, some nodes might not be caught, some might've progressed, so 80% is probably a better number haha.